pkg/kv/kvserver/replica.go

```diff
@@ -913,6 +913,18 @@ func (r *Replica) GetGCThreshold() hlc.Timestamp {
 	return *r.mu.state.GCThreshold
 }
 
+// ExcludeReplicaFromBackup returns whether the replica is to be excluded from a
+// backup.
+func (r *Replica) ExcludeReplicaFromBackup() bool {
```

Comment:
Can drop "Replica" from the method name, could maybe just use "ExcludeDataFromBackup" like everywhere else. Here and throughout.